README.md

```diff
@@ -76,7 +62,7 @@ The following packages are deprecated.
 | [**sensors_plus_tizen**](packages/sensors_plus) | ✔️ | ✔️ | ❌ | ❌ | No sensor hardware |
 | [**share_plus_tizen**](packages/share_plus) | ⚠️ | ⚠️ | ❌ | ❌ | No SMS or e-mail app |
 | [**shared_preferences_tizen**](packages/shared_preferences) | ✔️ | ✔️ | ✔️ | ✔️ |
-| [**sqflite_tizen**](packages/sqflite) | ✔️ | ✔️ | ✔️ ️| ✔️ |
+| [**sqflite_tizen**](packages/sqflite) | ✔️ | ✔️ | ✔️ | ✔️ |
```

Comment:
Question. Are there any changes?

Reply:
Yes, although it's hard to notice.

![image](https://user-images.githubusercontent.com/40190588/147196728-8871fe6f-6212-4ef2-bb73-8a0c45378735.png)